modules/platforms/dotnet/Apache.Ignite.Core.Tests/ApiParity/ClusterParityTest.cs

```diff
@@ -37,7 +37,8 @@ public class ClusterParityTest
             "baselineAutoAdjustEnabled",
             "baselineAutoAdjustTimeout",
             "baselineAutoAdjustStatus",
-            "clientReconnectFuture"
+            "clientReconnectFuture",
+            "readOnly"
```

Comment:
Why this property marked as unneeded ?

Reply:
I moved to MissingMembers and create ticket https://ggsystems.atlassian.net/browse/GG-23363 for implementation.